plat/xilinx/zynqmp/aarch64/zynqmp_common.c

```diff
@@ -40,10 +40,19 @@
  * configure_mmu_elx() will give the available subset of that,
  */
 const mmap_region_t plat_arm_mmap[] = {
-	{ DEVICE0_BASE, DEVICE0_BASE, DEVICE0_SIZE, MT_DEVICE | MT_RW | MT_SECURE },
-	{ DEVICE1_BASE, DEVICE1_BASE, DEVICE1_SIZE, MT_DEVICE | MT_RW | MT_SECURE },
-	{ CRF_APB_BASE, CRF_APB_BASE, CRF_APB_SIZE, MT_DEVICE | MT_RW | MT_SECURE },
-	{0}
+	{
+		DEVICE0_BASE, DEVICE0_BASE, DEVICE0_SIZE,
+		MT_DEVICE | MT_RW | MT_SECURE | MT_EXECUTE_NEVER,
+	},
+	{
+		DEVICE1_BASE, DEVICE1_BASE, DEVICE1_SIZE,
+		MT_DEVICE | MT_RW | MT_SECURE | MT_EXECUTE_NEVER,
+	},
+	{
+		CRF_APB_BASE, CRF_APB_BASE, CRF_APB_SIZE,
+		MT_DEVICE | MT_RW | MT_SECURE | MT_EXECUTE_NEVER,
+	},
+	{ 0 }
```

Comment:
This change has no functional effect; device memory is always XN. Are you just trying to emphasise this? Since there is no commit message, it might make the reader think this improves security when it doesn't.